apps/system-e2e/src/integration/service-portal/homepage.spec.ts

```diff
@@ -1,10 +1,10 @@
+const phoneNumber = Cypress.env('PHONE_NUMBER')
+
 describe('Home page', () => {
-  before(() => {
-    cy.ensureLoggedIn({ url: '/minarsidur/' })
```

Comment:
I think we still need to ensure the Cognito login is taken care of in environments where that is present.

Reply:
Swapped `ensureLoggedIn` out for `cognitoLogin` and made it run before each test.